rgw/admin/bucket.go

```diff
@@ -19,15 +20,16 @@ type Bucket struct {
 		DataExtraPool string `json:"data_extra_pool"`
 		IndexPool     string `json:"index_pool"`
 	} `json:"explicit_placement"`
-	ID        string `json:"id"`
-	Marker    string `json:"marker"`
-	IndexType string `json:"index_type"`
-	Owner     string `json:"owner"`
-	Ver       string `json:"ver"`
-	MasterVer string `json:"master_ver"`
-	Mtime     string `json:"mtime"`
-	MaxMarker string `json:"max_marker"`
-	Usage     struct {
+	ID           string     `json:"id"`
+	Marker       string     `json:"marker"`
+	IndexType    string     `json:"index_type"`
+	Owner        string     `json:"owner"`
+	Ver          string     `json:"ver"`
+	MasterVer    string     `json:"master_ver"`
+	Mtime        string     `json:"mtime"`
+	CreationTime *time.Time `json:"creation_time"`
```

Comment:
Looking at the following response from one of the test runs,
```
{"bucket":"test","num_shards":11,"tenant":"","zonegroup":"b701c2b3-5fb0-492d-9966-d774d8aad683","placement_rule":"default-placement","explicit_placement":{"data_pool":"","data_extra_pool":"","index_pool":""},"id":"9115881e-13d0-4cd0-a700-1b5424709afa.4200.1","marker":"9115881e-13d0-4cd0-a700-1b5424709afa.4200.1","index_type":"Normal","versioned":false,"versioning_enabled":false,"object_lock_enabled":false,"mfa_enabled":false,"owner":"admin","ver":"0#1,1#1,2#1,3#1,4#1,5#1,6#1,7#1,8#1,9#1,10#1","master_ver":"0#0,1#0,2#0,3#0,4#0,5#0,6#0,7#0,8#0,9#0,10#0","mtime":"2024-07-11T14:33:45.833704Z","creation_time":"2024-07-11T14:33:43.502753Z","max_marker":"0#,1#,2#,3#,4#,5#,6#,7#,8#,9#,10#","usage":{},"bucket_quota":{"enabled":false,"check_on_raw":false,"max_size":-1,"max_size_kb":0,"max_objects":-1}}
```

I think `Mtime` can also be of type `*time.Time`. But that's not intended here.